Sources/Core/Tasks/TaskFetchDecodedImage.swift

```diff
@@ -68,7 +68,7 @@ final class TaskFetchDecodedImage: ImagePipelineTask<ImageResponse> {
         if let response = response {
             send(value: response, isCompleted: isCompleted)
         } else if isCompleted {
-            send(error: .decodingFailed)
+            send(error: .decodingFailed(response?.container.data ?? Data()))
```

Comment:
@freak4pc, I was reworking this code for Nuke 11 and just noticed that it never attaches data. In this code block `response` is always `nil`.

Reply:
That's interesting. So where would we be able fetching the "right" Data from?

Reply:
I'll fix it in the patch release later today.

Reply:
The issue is that `ImageResponse` is always `nil` in this block – decoding failed.

Reply:
Fixed in [10.11.1](https://github.com/kean/Nuke/releases/tag/10.11.1).